sdk/ai/ci.yml

```diff
@@ -0,0 +1,42 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+    - main
+    - hotfix/*
+    - release/*
+    - restapi*
+  paths:
+    include:
+    - sdk/ai/
+    - sdk/core/    
+
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+    - restapi*
+  paths:
+    include:
+    - sdk/ai/
+    - sdk/core/    
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: ai
+    TestTimeoutInMinutes: 75
+    TestProxy: true
+    # This is a short term solution to create API review for python azure-ml package only when running pipeline manually
+    # Long term solution should be to have different versions on main branch and release branch for python package so APIView can have different revisions for each version.
+    # Tracking issue: https://github.com/Azure/azure-sdk-for-python/issues/29196
+    GenerateApiReviewForManualOnly: true
```

Comment:
FYI @needuv I'm not certain what the story was with `ml/ci.yml` to have this, but I just mirrored it in case. If not please add a correction.

Reply:
we'll revisit this post-ignite